security/certificate_types_descriptions/service-ca-certificates.adoc

```diff
@@ -27,6 +27,11 @@ The service CA expiration of 26 months is longer than the expected upgrade inter
 A manually-rotated service CA does not maintain trust with the previous service CA. You might experience a temporary service disruption until the pods in the cluster are restarted, which ensures that pods are using service serving certificates issued by the new service CA.
 ====
 
+[IMPORTANT]
+====
+Applications that use a different trust store, such as JavaKeyStore, need to be capable of accepting new CA certifications dynamically into it's trust store, or else when automated rotation occurs, pods will require a restart.
```

Comment:
I think it might be helpful to break this up in 2 sentences. Something like this maybe? 
```suggestion
Applications that use a different trust store, such as JavaKeyStore, need to be capable of accepting new CA certifications dynamically into it's trust store.  Otherwise, when automated rotation occurs, pods will require a restart.
```

Reply:
Note, the suggestion `.  Otherwise` has accidental two spaces between `.` and `O`